clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/config/KubernetesConfigurationProperties.groovy

```diff
@@ -42,7 +42,8 @@ class KubernetesConfigurationProperties {
     List<LinkedDockerRegistryConfiguration> dockerRegistries
     List<String> requiredGroupMembership
     Permissions.Builder permissions = new Permissions.Builder()
-    Boolean debug = false;
+    String namingStrategy = "annotations"
```

Comment:
Sorry let's call it "kubernetesAnnotations" to avoid overlap with other providers.